src/index.html

```diff
@@ -1,17 +1,118 @@
-<!doctype html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
-  </head>
-  <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
-  </body>
+<!DOCTYPE html>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport"content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML Form</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"> 
+    <fieldset class="section">
+      <legend>Personal information</legend>
+      <label class="form">
+        Surname:
+        <input type="text" autocomplete="off" 
+        name="surname"
+        >
+      </label>
+      <label class="form">
+          Name:
+        <input type="text" name="name"
+        autocomplete="off"
+        >
+      </label>
+      <label class="form">
+          How old are You?
+        <input type="number" 
+          name="age"
+          value="12"
+          minlength="1"
+          maxlength="100"
+        >
+      </label>
+      <label class="form">
+          Full date of birth:
+        <input type="date" name="birthday">
+      </label>
+      <label>
+          I accept the term of the agreement
+        <input type="checkbox" 
+          name="agree" 
+          required
+        >
+      </label>
+    </fieldset>
+    <fieldset class="section">
+      <legend>Registration</legend>
+      <label class="form inline">
+          E-mail
```

Comment:
![Screenshot from 2023-02-06 20-49-55](https://user-images.githubusercontent.com/33029864/217059289-d8ed84e4-c5dc-49db-abe5-3705e820da63.png)
Absent semicolons your solution it can be reason.